src/test/java/wolox/training/TrainingApplicationTests.java

```diff
@@ -8,4 +10,8 @@
 @SpringBootTest
 public class TrainingApplicationTests {
 
+    @Test
+    public void dummy() {
```

Comment:
It would be nicer to change the name to contextLoad or something like that because that's the purpose of this test, right?